bitrise.yml

```diff
@@ -1687,6 +1687,77 @@ workflows:
         - maximum_test_repetitions: 10
         - xcodebuild_options: '"COMPILER_INDEX_STORE_ENABLE=NO" "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO" "-parallel-testing-enabled" "YES" "-parallel-testing-worker-count" "2"'
     - deploy-to-bitrise-io@2.10.0: {}
+
+  Swift_6_error_count:
+    before_run:
+    - 1_git_clone_and_post_clone
+    - 2_certificate_and_profile
+    - 3_provisioning_and_npm_installation
+    steps:
+    - xcode-build-for-test@3:
+        inputs:
+        - scheme: Fennec
+        - configuration: Fennec_Testing
+        - project_path: /Users/vagrant/git/firefox-ios/Client.xcodeproj
+        - xcodebuild_options: '"COMPILER_INDEX_STORE_ENABLE=NO" "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO"'
+        - destination: platform=iOS Simulator,name=iPhone 16,OS=18.2
+    - script@1.2.1:
```

Comment:
As per @FilippoZazzeroni we could add these steps to the regular pipeline so that we check this for each PR and not in a cron or ad-hoc job